configs/hrformer/readme.md

```diff
@@ -0,0 +1,59 @@
+# HRFormer
+
+[HRFormer: High-Resolution Transformer for Dense Prediction](https://arxiv.org/pdf/2110.09408.pdf)
+
+## Introduction
+
+<!-- [BACKBONE] -->
+
+<a href="https://github.com/HRNet/HRFormer">Official Repo</a>
+
+<a href="https://github.com/open-mmlab/mmsegmentation/blob/v0.21.0/mmseg/models/backbones/hrformer.py#L562">Code Snippet</a>
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+We present a High-Resolution Transformer (HRFormer) that learns high-resolution representations for dense prediction tasks, in contrast to the original Vision Transformer that produces low-resolution representations and has high memory and computational cost. We take advantage of the multi-resolution parallel design introduced in high-resolution convolutional networks (HRNet), along with local-window self-attention that performs self-attention over small non-overlapping image windows, for improving the memory and computation efficiency. In addition, we introduce a convolution into the FFN to exchange information across the disconnected image windows. We demonstrate the effectiveness of the High- Resolution Transformer on both human pose estimation and semantic segmentation tasks, e.g., HRFormer outperforms Swin transformer by 1.3 AP on COCO pose estimation with 50% fewer parameters and 30% fewer FLOPs.
+
+<!-- [IMAGE] -->
+<div align=center>
+<img src="https://user-images.githubusercontent.com/41846794/160131437-46a06d14-549b-41c4-9fc0-e4785fb63468.png" width="70%"/>
+</div>
+
+
+```bibtex
+@article{YuanFHLZCW21,
+  title={HRFormer: High-Resolution Transformer for Dense Prediction},
+  author={Yuan, Yuhui and Fu, Rao and Huang, Lang and Lin, Weihong and Zhang, Chao and Chen, Xilin and Wang, Jingdong},
+  booktitle={NeurIPS},
+  year={2021}
+}
+```
+
+## Usage
+
+To use the pre-trained models from the [official repository](https://github.com/HRNet/HRFormer), it is necessary to convert keys.
```

Comment:
In new version of MMSegmentation, we would provide well-transferred pretrained models for better use experience. Please follow this [PR: Provide URLs of Swin Transformer pretrained models](https://github.com/open-mmlab/mmsegmentation/pull/1389). 

To do (1): pull down well-transferred pretrained models and GENERATE HASH CODE for their `.pth` file using `process_checkpoint` from [gather_models](https://github.com/open-mmlab/mmsegmentation/blob/master/.dev/gather_models.py#L25). (2) change README.md in this line (refer to readme in swin). (3) I would upload these mmseg-style pretrained models and give you urls. (4) change these urls in hrformer config file.